karma.conf.js

```diff
@@ -0,0 +1,95 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = (config) => {
+  config.set({
+    basePath: '',
+
+    frameworks: ['mocha', 'sinon', 'chai'],
+
+    files: [
+      'test/react/components/*',
+    ],
+
+    webpack: {
+      externals: {
+        sinon: true,
+      },
+      module: {
+        rules: [
+          {
+            test: /\.jsx?$/,
+            loader: 'babel-loader',
+            include: [
+              path.join(__dirname, 'src'),
+              path.join(__dirname, 'test'),
+            ],
+            query: {
```

Comment:
Yeah, a webpack / build refactor change would be nice at some point.  No need for it to be done now though.